Modix/ClientApp/src/components/UserLookup/UserProfile.vue

```diff
@@ -87,6 +98,7 @@ export default class UserProfile extends ModixComponent
     async mounted()
     {
         await store.retrieveRoles();
+        this.messages = await UserService.getMessageCountPerChannel(this.$store.state.modix.user.userId);
```

Comment:
I think this will only ever pull message stats for the logged-in user instead of the user whose info we're trying to display. I think we can fix this by moving the service call into a Watch on `user` and call it with `user.id`.

```ts
import { Component, Prop, Watch } from 'vue-property-decorator';    <-- added Watch to the imports

// ...

@Watch('user')
async setMessages(): Promise<void>
{
    if (this.user && this.user.id)
    {
        this.messages = await UserService.getMessageCountPerChannel(this.user.id);
    }
}

async mounted()
{
    await store.retrieveRoles();
}
```

Reply:
oh right at first those IDs are the same and I tested it in empty guild so I had no other user to test
(now I found out that it can search for bots too)

fixed